generators/php/math.js

```diff
@@ -150,7 +150,7 @@ Blockly.PHP['math_number_property'] = function(block) {
         Blockly.PHP.ORDER_EQUALITY],
     'ODD': [' % 2 == 1', Blockly.PHP.ORDER_MODULUS,
         Blockly.PHP.ORDER_EQUALITY],
-    'WHOLE': ['is_int()', Blockly.PHP.ORDER_MODULUS,
+    'WHOLE': ['is_int(' + number_to_check + ')', Blockly.PHP.ORDER_MODULUS,
```

Comment:
Note the error from https://github.com/RichDoherty/blockly/runs/4128824474?check_suite_focus=true#step:8:95:

Size of build/blockly_compressed.js at 920066 compared to previous Size of build/blocks_compressed.js at 82056 compared to previous Size of build/blockly_compressed.js.gz at 168774 compared to previous Size of build/blocks_compressed.js.gz at 16591 compared to previous Evaluation failed: **ReferenceError: number_to_check is not defined**

`number_to_check` will not be defined at this point.  We need to handle this case differently because we're no longer just appending a suffix to `number_to_check` as we do for `ODD`, `EVEN`, `POSITIVE`, and `NEGATIVE`.

Also, the reason there was a problem for whole before is in PHP, `x % 1` will be 0 for all numbers x (including the approximation `M_PI` to &pi;) so `M_PI % 1 == 0` would evaluate to `true`, thereby incorrectly classifying `M_PI` as a whole number.